spec/example_app/app/assets/images/logo.svg

```diff
@@ -0,0 +1,16 @@
+</mask>
+<g mask="url(#mask0)">
```

Comment:
What does this `url(#mask0)` thing do?

Reply:
It defines which shape to use as a mask (in this case, the gradient image is being masked by the text)